timemachine/cpp/src/kernels/k_barostat.cuh

```diff
@@ -0,0 +1,153 @@
+#pragma once
+
+#include "k_fixed_point.cuh"
+
+void __global__ rescale_positions(
+    const int N,                                     // Number of atoms to shift
+    double *__restrict__ coords,                     // Coordinates
+    const double *__restrict__ length_scale,         // [1]
+    const double *__restrict__ box,                  // [9]
+    double *__restrict__ scaled_box,                 // [9]
+    const int *__restrict__ atom_idxs,               // [N]
+    const int *__restrict__ mol_idxs,                // [N]
+    const int *__restrict__ mol_offsets,             // [N]
+    const unsigned long long *__restrict__ centroids // [N*3]
+) {
```

Comment:
Worth adding a short docstring-like description of what these functions do?

Reply:
Added docstrings in [c1f9bc0](https://github.com/proteneer/timemachine/pull/1101/commits/c1f9bc0f60da1d8bd30bcae7048644885c8b0364)